projects/plugins/crm/.gitattributes

```diff
@@ -35,15 +44,19 @@ vendor/jetpack-autoloader/**                    production-include
 .gitignore                                      production-exclude
 .phpcs.dir.xml                                  production-exclude
 babel.config.js                                 production-exclude
-build/**/*.js.map                               production-exclude
 changelog/**                                    production-exclude
 composer.lock                                   production-exclude
 package.json                                    production-exclude
 phpunit.xml.dist                                production-exclude
 README.md                                       production-exclude
-src/js/**                                       production-exclude
+/sass/**                                        production-exclude
 tests/**                                        production-exclude
 /vendor/automattic/jetpack-autoloader/**        production-exclude
 /vendor/automattic/jetpack-changelogger/**      production-exclude
 /vendor/automattic/jetpack-composer-plugin/**   production-exclude
-webpack.config.js                               production-exclude
\ No newline at end of file
+webpack.config.js                               production-exclude
+**/css/*.min.css.map                            production-exclude
+**/css/**/*.min.css.map                         production-exclude
+**/js/*.min.js.map                              production-exclude
+**/js/**/*.min.js.map                           production-exclude
+**/js/*.min.js.LICENSE.txt                      production-exclude
```

Comment:
You might want to include these LICENSE.txt files in the distributed plugin, as licenses on bundled JS packages may require that the notices be kept with the distribution.

OTOH, I think currently the only things being included in any of these are `/*!` comments from stuff in this repo. If you change those comments to remove the `!`, the files might go away entirely.

Reply:
Ah good to know. That could be something we can follow up with (changing comments), possibly even next week, but adding the license.txt line to production-include to catch it anyway.